### PR TITLE
Prep 1.1.0 release

### PR DESCRIPTION
**Pull request by viveklak**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,14 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+---
+
 #### New Resources
+## 1.1.0 (2020-04-20)
```
Comment (mikhailshilkov) [inline CHANGELOG.md:12]: Swap the two lines above

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,12 +6,14 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
+---
```
Comment (mikhailshilkov) [inline CHANGELOG.md:9]: Nit: add `(None)` above this line

Comment (viveklak) [reply CHANGELOG.md:11]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (viveklak) [reply CHANGELOG.md:13]: Done.

**Review by mikhailshilkov — APPROVED** [review]

**Pull request merged**